packages/calcite-components/src/components/popover/popover.stories.ts

```diff
@@ -124,18 +124,20 @@ export const flipPlacements_TestOnly = (): string => html`
 `;
 
 export const scaleConsistencyPopoverHeadingActionSlottedIcon_TestOnly = (): string => html`
-  <div style="width: 400px;">
-    ${referenceElementHTML}
-    <calcite-popover
-      heading="Dreams didn't make us kings. Dragons did. 🐉"
-      reference-element="reference-element"
-      placement="auto"
-      open
-      closable
-      scale="m"
-    >
-      ${contentHTML}
-    </calcite-popover>
+  <div style="width: 800px; height:800px;">
```

Comment:
Sidebar: do you think adding a shared set of storybook decorators (similar to [this](https://www.chromatic.com/docs/troubleshooting-snapshots/#use-fixed-height-wrappers-for-portal-components) or [this](https://www.chromatic.com/docs/troubleshooting-snapshots/#why-isnt-my-modal-or-dialog-captured)) to normalize the dimensions of similar stories would be useful?

Reply:
Yes that would be very helpful. We already do variations of this for some components.

Reply:
Sweet. I'll create an issue for this.